README.md

```diff
@@ -71,7 +71,7 @@ You need the following tools:
   - sbt, we recommend the [sbt-extras](https://github.com/paulp/sbt-extras) runner
   script. It provides sensible default jvm options (stack and heap size).
   - curl (for `./pull-binary-libs.sh`, used by the sbt / ant build).
-  - Apache Ant (version 1.9.3 or above) if you need to use the (deprecated) ant build.
+  - Apache Ant (version 1.9.x, minimum 1.9.3; Ant 1.10+ doesn't work on Java 6) if you are using the ant build.
```

Comment:
Easier to decide from?

* Apache Ant if your are using the ant build.
  * Version >= 1.9.3 if using Java 8
  * Version >= 1.9.3, < 1.10 if using Java 6